docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -69,36 +98,75 @@ NOTE: The `keycloakConfig` field is partially implemented in link:{WFM-RC-CoreUR
                     "use-resource-role-mappings": true
                   }
 ----
+NOTE: Ensure that these values corresponds to your client configuration within your realm. For more information,
+see the {Keycloak} link:{WFM-RC-KeycloakURL}securing_apps/topics/oidc/nodejs-adapter.html[Node.js Adapter documentation].
+
+IMPORTANT: Filling in the values in the keycloakConfig also enables {Keycloak} in the demo server application
```

Comment:
I will expect us to at least say what I need to fill. There is a lots of background info in this docs that aren't stritly needed, but we missing the esential stuff - what is part of the solution and what should not be changed as it may break things really bad. 

Things like Realm needs to be also explained. They do not need to use RainCatcher realm or the same clients if they have one but then developer will take care to replicate setup as it is.
This is the main thing the most of the keycloak aware people will ask - how I can adapt my realm or change realm to not break it and still be supported. 

Rest of the documentation is good to have but IMHO provides minimal to zero useful info for person who want to build RainCatcher on top of the keycloak.